lab1_update.js

```diff
@@ -0,0 +1,164 @@
+/*jshint strict:false */
+'use strict';
+
+/* LAB 1: A Trip to Woodland Park Zoo
+
+ Welcome to Lab 1 :)
+
+ Be sure to read all the comments!
+
+ All of the instructions are inline with the assignment below. Look for the
+ word TODO in comments.  Each TODO will have a description of what is
+ required.
+
+ To run this file, type the following (in the terminal):
+
+    node lab1.js
+
+ and then press enter/return.
+
+ Instructions for turning this lab in are in the assignment description in
+ Canvas.
+
+ I'm happy to answer any questions on Slack.
+
+*/
+
+/* ----------------- Helper ------------------------------------------------
+ We've implemented a function that will help you sanity-check your code.
+*/
+
+function assert(expression, failureMessage) {
+  if (!expression) {
+    console.log('assertion failure: ', failureMessage);
+  }
+}
+
+/*
+ An assertion is something we expect to be true about the state of the
+ program at the point where the assertion is made.
+
+ Remember, state is all the variables that we've declared and their current
+ values.
+
+ Below are some examples for how to use the assert method. Tip: once you've run
+ this file a couple times to see how they work, comment them out.
+ That will help later when you run jscs.
+*/
+
+// assert(1 === 1);
+// assert(1 === 2, 'this is an assertion failure example. 1 === 2');
+
+/*------------------Assertions------------------------------------------------
+ TODO: 8 points
+ Invoke assert twice. (In other words, create two assertions like the ones above.)
+ Use any boolean expressions that you like, but preferably
+ zoo-themed.  Make one pass and one fail. In the failure message, describe why
+ it failed.
+*/
+
+assert('lion' == 'big cat');
+assert ('lion' === 'cheetah');
+
+The second assertion fails because although a lion (and a cheetah) are both big
+cats, a lion and a cheetah are not the same species.
+
```

Comment:
If this is your assertion message (the lion and the cheetah are not the same species), it needs to be passed in as an argument to the assert function. The assert function takes this for arguments:
assert (expression, message)
For example:
assert (true === false, "true isn't false, you silly!")
When you just write a sentence out here, without commenting it out, the intrepreter reads that as code and doesn't know what to do with it.